detekt.yml

```diff
@@ -0,0 +1,349 @@
+build:
+  maxIssues: 10
+  weights:
+    complexity: 2
+    formatting: 1
+    LongParameterList: 1
+    comments: 1
+
+processors:
+  active: true
+  exclude:
+  # - 'FunctionCountProcessor'
```

Comment:
Please remove these if we don't need them 🙏

Reply:
@markgravity You mean the comment, right?

Reply:
@lydiasama yes, as we disable them so let's remove them 🙋

Reply:
Updated at 4f71c1ed